App.tsx

```diff
@@ -232,7 +232,7 @@ class LoadingView extends Component<LoadingProps, LoadingState> {
   }
 }
 
-const ZecwalletTheme = {
+const Theme = {
```

Comment:
hmm.. this was the only change I really wanted to make in this file. 

My editor wants to conform to a very common style in JS-world of putting spaces inside object literals. I could go either way. Can we format tsx the way I did here (the normal, default way these days). Or, should I revert the style changes?

I vote we change the style to what I have here?